### PR TITLE
Documentation update

### PR DESCRIPTION
**Pull request by DJCrapsody**
Changes include:
Replacing horizontal line with spaces because the lines look really ugly.
Documenting rebase of fork to help merge conflicts before doing a PR
Includes changes from #445 without commit badge (do we really _**need**_ it?)

Closes #445

Discussion for #449 to be concluded so as to allow checking against checksum without downloads section

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,21 +24,6 @@ GoIV is an android app made for Pokémon GO that allows users to seamlessly calc
...
-**GoIV (Online)**  
-MD5 - F96E4F3E51E8B77D507BD53CE04FAD46  
-SHA1 - 9AA4448FD64A21F0D4364046F4BD36EF37C804FB  
-SHA-256 - 22214A2DB976C0D1BD6702B8FC5D51D23AAC7EFA937C8A68BCCBB772EFBFE8B1  
-
-**GoIV (Offline)**  
-MD5 - 66998267C373CAB90FA0CEC27FF9AB2C  
-SHA1 - B73A092FC8727E1301F8C590E71ADC4DE6273380  
-SHA-256 - 64CCFEA3434BE407DC49841DECFE787E24DC13F6BBD4E73C2CF98F5A44F53274  
-
```
Comment (Blaisorblade) [inline README.md]: Fixed it enough.

Comment (Blaisorblade) [inline README.md]: And... cherry-picked!

**Comment by Blaisorblade** [comment]
I've cherry-picked the commits I'm happy with; I don't agree with short docs on rebasing, and I'm not sure about lines—and especially about adding spaces there—they're invisible, what's the point?

So closing for now.

**Comment by DJCrapsody** [comment]
@Blaisorblade They're not invisible, markdown merges multiple break lines (unless using html br tags OR using nbsp.

**Comment by Blaisorblade** [comment]
That's the sort of thing to be mentioned in a commit message. `<br />` would be clearer for that. And I'm not sure that's worth styling by hand this way. But this is really a detail...

**Comment by DJCrapsody** [comment]
We'll use br tags then I guess

**Comment by Blaisorblade** [comment]
To be sure, I haven't said that's fine, just less bad. If you really hate them, I'd just drop the horizontal lines.

**Comment by DJCrapsody** [comment]
Would prefer to drop the horizontal lines, but if reverting to no spaces or lines seems okay, then we can revert it back. Was trying out to see if the lines look fine anyway, the spaces are okay but takes additional effort.

**Pull request closed without merging**